scripts/quickstart_tooling_dpg/main.py

```diff
@@ -5,14 +5,16 @@
 from jinja2 import Environment, FileSystemLoader
 from subprocess import check_call
 import time
-from typing import Any, Dict
+from typing import Any
+import json
 
 _LOGGER = logging.getLogger(__name__)
 
 _TEMPLATE = Path(__file__).resolve().parent / "template"
 _TEMPLATE_TESTS = Path(__file__).resolve().parent / "template_tests"
 _TEMPLATE_SAMPLES = Path(__file__).resolve().parent / "template_samples"
 _TEMPLATE_CI = Path(__file__).resolve().parent / "template_ci"
+_CONFIG_FILE = Path(__file__).resolve() / "../../../swagger_to_sdk_config_dpg.json"
```

Comment:
I think, line 17, should be ,

```
_CONFIG_FILE = Path(__file__).resolve().parent / "../../../swagger_to_sdk_config_dpg.json"
```

parent should be appended. Why? because without adding .parent this points to file main.py, not a folder. 

Error Recieved: 
Input
```
python ./scripts/quickstart_tooling_dpg/main.py --output-folder ./sdk/loadtestservice/azure-developer-loadtest --input-file /mnt/c/Users/niveditjain/Desktop/loadtesting.json --security-scope https://loadtest.azure-dev.com/.default --package-name azure-developer-loadtestservice --package-pprint-name "Azure Developer LoadTestService" --client-name LoadTestClient
```
Output: 
```
INFO:__main__:Build start: azure-developer-loadtestservice
INFO:__main__:Building swagger readme
Traceback (most recent call last):
  File "./scripts/quickstart_tooling_dpg/main.py", line 186, in <module>
    main(**parameters)
  File "./scripts/quickstart_tooling_dpg/main.py", line 126, in main
    build_package(**kwargs)
  File "./scripts/quickstart_tooling_dpg/main.py", line 105, in build_package
    autorest_cmd = f'autorest {swagger_readme} {get_autorest_version()} '
  File "./scripts/quickstart_tooling_dpg/main.py", line 80, in get_autorest_version
    with open(_CONFIG_FILE, 'r') as file_in:
NotADirectoryError: [Errno 20] Not a directory: '/mnt/c/Users/niveditjain/Desktop/azure-sdk/scripts/quickstart_tooling_dpg/main.py/../../../swagger_to_sdk_config_dpg.json'
```
Error was resolved by rewriting the line 17 as,
```
 _CONFIG_FILE = Path(__file__).resolve().parent / "../../swagger_to_sdk_config_dpg.json"
```